test/unit/presenters/publishing_api/case_study_presenter_test.rb

```diff
@@ -64,7 +64,7 @@ def present(edition)
       presented_item.content[:details].delete(:body)
     )
     assert_equal expected_content[:details], presented_item.content[:details].except(:body)
-    assert_equal expected_links, presented_item.links
+    assert_hash_includes presented_item.links, expected_links
```

Comment:
Aren't we weakening the test a bit with this? Before it was testing that all of the expected links and only the expected links were being sent and now it is testing that at least the expected links are being sent. We could be sending others that we don't expect and this test still passes?